src/main/java/mate/academy/spring/config/WebConfig.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.spring.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@EnableWebMvc
+@Configuration
+@ComponentScan(basePackages = "mate.academy.spring.controller")
+public class WebConfig {
```

Comment:
does it work without `implements WebMvcConfigurer`?

Reply:
Honestly, i didn't check...